src/components/AstronautsPage/components/AstronautsList/components/AstronautsItem/components/AstronautsItemImage/AstronautsItemImage.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { CardMedia } from '@mui/material';
+import { astronautsData } from '../../../../../../mocks/astronautsData';
+
+export const AstronautsItemImage = () => {
+  return (
+    <CardMedia
+      component="img"
+      height="240"
```

Comment:
1. can avoid using return here
2. move height to scss

Reply:
done